src/GT.Core/DTO/Impl/ListingDTO.cs

```diff
@@ -47,5 +47,12 @@ public class ListingDTO : IGTDataTransferObject
 		/// </summary>
 		[StringLength(100)]
 		public string? ExperienceLevel { get; set; }
+
+		/// <summary>
+		/// Maps to an ApplicationDeadline name in the database.
+		/// </summary>
+		public DateTime? ApplicationDeadline { get; set; }
+
+		public string? CompanyLogoId { get; set; }
```

Comment:
Should perhaps have a StringLength limit of 450?

Reply:
Fixed limit